src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/welcome-to-premium/page.tsx

```diff
@@ -35,6 +36,12 @@ export default async function WelcomeToPremiumPage() {
     user: session.user,
   };
 
+  // If the current user is a subscriber and their OneRep profile is not
+  // activated: Most likely we were not able or failed to kick-off the
+  // auto-removal process.
+  // Let’s make sure the users OneRep profile is activated:
+  await activateAndOptoutProfile(profileId);
```

Comment:
Does this need to be wrapped in a `try..catch` block?
IIUC, the `activeAndOptoutProfile()` fn has this:
```js
  if (profileStatus === "inactive") {
    await activateProfile(profileId);
    await optoutProfile(profileId);
  } else {
```

and both the `activateProfile()` and `optoutProfile()` functions will potentially throw errors.

Reply:
Good point! Added in [24f3b68](https://github.com/mozilla/blurts-server/pull/3705/commits/24f3b6848a0d03540baa79f72b09bb7187056cbc).